src/components/SourcesTable/SourcesTable.js

```diff
@@ -1,7 +1,9 @@
 import React, { useEffect, useReducer } from 'react';
 import { useDispatch, useSelector, shallowEqual } from 'react-redux';
 import { useHistory } from 'react-router-dom';
-import { Table, TableHeader, TableBody, sortable, wrappable } from '@patternfly/react-table';
+
+import { Table, TableHeader, TableBody, sortable, wrappable } from '@patternfly/react-table/dist/esm/index';
```

Comment:
This is not required. Webpack will use the ESM assets anyway plus the table package does not treeshake react-core anyway :shrug: wont help.

Reply:
Yeah, but I had to use 'cjs' paths in tests for checking table icons. This makes it more transparent.

Reply:
But yeah, you are right, the mixing is not a big price.